app/src/main/java/com/yml/womensafety/AccelerometerService.kt

```diff
@@ -32,6 +32,9 @@ class AccelerometerService : Service(), SensorEventListener {
     private var lastY = 0f
     private var lastZ = 0f
     private val shakeThreshold = 15f
+    private var broadcastAction = "com.yml.womensafety.ACTION"
```

Comment:
Constant names should be in caps separated by underscore.

Reply:
done